src/components/Dialog.js

```diff
@@ -14,17 +37,5 @@ export default function Dialog(type, options) {
     }
   }
 
-  if (type == 'Open') {
-    return libui.UiDialogs.openFile(CURRENT_WINDOW);
-  } else if (type == 'Save') {
-    return libui.UiDialogs.saveFile(CURRENT_WINDOW);
-  } else if (type == 'Message') {
-    libui.UiDialogs.msgBox(CURRENT_WINDOW, options.title, options.description);
```

Comment:
You seem to passing as arguments here, but receiving them in the function as an unpacked object. I would prefer simple arguments.